sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -131,6 +133,14 @@ object SQLMetrics {
     acc
   }
 
+  def createTimestampMetric(sc: SparkContext, name: String): SQLMetric = {
```

Comment:
createTimelineMetric

Reply:
Copy, changes in SQLMetrics has been reverted in next commit.